modules/Quiz/resources/quiz.js

```diff
@@ -29,6 +29,7 @@
         multiStreamWelcomeSkip:false,
         relatedStreamChanging:false,
         IVQVer:'IVQ-2.41.rc2',
+        ivqShowScreenMode: false,
```

Comment:
ShowScreen is too generic. This is a vertical-block screen. Let's try 'mobileWarningLayer' or something that will imply the meaning of the screen